libs/platform/experience/carousel-navigation/carousel-navigation.component.ts

```diff
@@ -377,9 +375,17 @@ export class CarouselNavigationComponent
   }
 
   /**
-   * Handles the click on an indicator.
+   *
+   * @param @deprecated use handleIndicator instead
    */
   protected handleIndicatorClick(e: Event): void {
+    this.handleIndicator(e);
+  }
+
+  /**
+   * Handles the click on an indicator.
+   */
+  protected handleIndicator(e: Event): void {
```

Comment:
**handleIndicatorInput?** or **onIndicatorInput**, or any other more specific name for this method. And this can be handled not by click only, keyboard navigation is doing the same, check the description of it